specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/HybridCompute.json

```diff
@@ -713,27 +613,28 @@
             "pattern": "[a-zA-Z0-9-_\\.]",
             "minLength": 1,
             "maxLength": 54,
-            "description": "The name of the machine containing the extension."
+            "description": "The name of the hybrid machine."
           },
           {
-            "name": "extensionName",
-            "in": "path",
-            "required": true,
+            "name": "$expand",
+            "in": "query",
+            "required": false,
             "type": "string",
-            "description": "The name of the machine extension."
-          },
-          {
-            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
-          },
-          {
-            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+            "description": "The expand expression to apply on the operation.",
+            "enum": [
+              "instanceView"
+            ],
+            "x-ms-enum": {
+              "name": "InstanceViewTypes",
+              "modelAsString": true
+            }
           }
         ],
         "responses": {
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/MachineExtension"
+              "$ref": "#/definitions/Machine"
```

Comment:
so we have two different data models we'll return, depending on the value of the $expand query param. I'm not 100% certain how we're supposed to model that here. Compute should have something in their manifest that we could reference